### PR TITLE
feat(ui): trigger analysis

### PR DESCRIPTION
**Pull request by fsamin**

### CODE REVIEW

File: ui/src/app/views/projectv2/explore/trigger-analysis/trigger-analysis.component.ts [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+
+  splitRepository(repo: string): { vcs: string, repo: string } {
+    const splitted = repo.split('/');
+    return {
+      vcs: splitted.splice(0, 1)[0],
+      repo: splitted.join('/')
+    };
+  }
+
+  splitWorkflow(repo: string): { vcs: string, repo: string, workflow: string } {
```
Comment (sguiheux) [inline ui/src/app/views/projectv2/explore/trigger-analysis/trigger-analysis.component.ts:97]: Not used

**Review by sguiheux — CHANGES_REQUESTED** [review]

File: ui/src/app/views/projectv2/explore/trigger-analysis/trigger-analysis.component.ts (hunk 1 above)
Comment (fsamin) [reply ui/src/app/views/projectv2/explore/trigger-analysis/trigger-analysis.component.ts]: fixed

**Review by sguiheux — APPROVED** [review]

**Comment by ovh-cds** [comment]
CDS Report build-cds#21034.0 ✔ 
* 
  * Build and Package All -   
  * Build and Package Contrib Action ✔ 
  * Build and Package Contrib Code Plugin ✔ 
  * Build and Package Contrib Integrations ✔ 
  * Build and Package Engine, Worker, CLI and Doc ✔ 
  * Build and Package UI ✔ 
  * Build Debpacker ✔ 
  * Build SMTPMock ✔ 
  * Lint UI Translate ✔ 
  * Test CLI ✔ 
  * Test Contrib ✔ 
  * Test Engine ✔ 
  * Test SDK ✔ 
  * Test UI ✔

**Comment by ovh-cds** [comment]
CDS Report integration-cds#21034.0 ✘  
* Integration
  * Docker Compose Tests ✔ 
  * Standalone Tests ✔ 
  * Standalone Tests v1 ✘  
  * Standalone Tests v2 ✘

**Comment by ovh-cds** [comment]
CDS Report integration-cds#21034.1 ✔ 
* Integration
  * Docker Compose Tests ✔ 
  * Standalone Tests ✔ 
  * Standalone Tests v1 ✔ 
  * Standalone Tests v2 ✔

**Pull request merged**